tests/test-pull-corruption.sh

```diff
@@ -55,7 +55,16 @@ do_corrupt_pull_test() {
 
 # FIXME - ignore errors here since gjs in RHEL7 has the final
 # unrooting bug
-gjs $(dirname $0)/corrupt-repo-ref.js ${repopath} main || true
+
+# super hacky fix to support both installed and uninstalled tests. we'll
+# probably want to just enable it for one of them.
+if [ -f /lib64/libostree-1.so ]; then
```

Comment:
Actually, a better condition might be to just check if $BUILDDIR exists. Otherwise, we'll be using the installed SO even if it's not the one we just built.